data/lm/generate_lm.py

```diff
@@ -1,63 +1,135 @@
+import argparse
 import gzip
 import io
 import os
 import subprocess
-import tempfile
-
 from collections import Counter
 from urllib import request
 
-def main():
-  # Grab corpus.
-  url = 'http://www.openslr.org/resources/11/librispeech-lm-norm.txt.gz'
+import tqdm
```

Comment:
We already have a dependency against `progressbar2`, can we avoid multiplying the dependencies for the same feature?